src/types/enums.ts

```diff
@@ -12,3 +12,10 @@ export enum Severity {
   WARNING = 'Warning',
   INFO = 'Info'
 }
+
+export enum Part {
```

Comment:
Part er ganske abstrakt i denne sammenhengen, bruk gjerne noe mer konkret
```suggestion
export enum RequestParameter {
```